test/FunctionalTests/FunctionalTestBase.cs

```diff
@@ -26,9 +26,9 @@ namespace Grpc.AspNetCore.FunctionalTests
 {
     public class FunctionalTestBase
     {
-        private VerifyNoErrorsScope _scope;
+        private VerifyNoErrorsScope? _scope;
```

Comment:
Why is _scope nullable? We always expect that it will be initialized via Setup right?

Reply:
Because it isn't set in the constructor. If someone did not call SetUp then it would be null.